### ISSUE TITLE
Search addons by highlighted words

### ISSUE THREAD

**Issue by RIMOPA** [ScratchAddons/ScratchAddons#4387]
### Is your request related to a problem? Please describe.

Sometimes it's hard to find addons, and I think some highlited words per addon is the best solution.

### Describe the solution you'd like

Every addon, after its tags, has a array with a few highited words
Example:
` "name": "2D color picker",`
`  "description": "Replaces saturation and brightness sliders in the costume editor with a 2D color picker. Hold Shift while dragging the cursor to change the values on a single axis.",`
`  "tags": ["editor", "costumeEditor"],`
`  "highwords": ["pick", "color", "2D", "select", "saturation", "brightness"]`
`  "credits": [...`

### Additional context

![image](https://user-images.githubusercontent.com/97112832/157722340-9377b0e0-fd62-4cae-a0e0-9463709f77b1.png)

**Comment by mxmou** [comment]
This would probably be difficult to translate.

**Comment by RedGuy12** [comment]
@RIMOPA it would be difficult to translate because there would be no context for any of the strings

A keywords field was also discussed in #2799. Remember the effectiveness of this decreases when extended search is turned on.

I don't think extended search is needed tbh. With it off, I get much more relevant results. We also don't say anywhere that these filters are available.

**Comment by Chiroyce1** [comment]
I agree with @RedGuy12, search is already good enough now. You can find this specific addon by simply searching for `color` and scrolling down.

**Comment by RedGuy12** [comment]
> I agree with @RedGuy12, search is already good enough now. You can find this specific addon by simply searching for `color` and scrolling down.

I was saying extended search, which was enabled in #2799, is not needed. This issue is definitely a good idea. It would be hard to translate but I think it would be worth it.

**Comment by WorldLanguages** [comment]
I think search is good enough as-is.

**Comment by WorldLanguages** [comment]
There has been some discussion about this feature on #5375
